cli/types/login.d.ts

```diff
@@ -0,0 +1,13 @@
+export {}
+
+declare module 'auth0' {
+  interface ManagementClient<A = AppMetadata, U = UserMetadata> {
+    branding: {
+      getUniversalLoginTemplate: () => Promise<{ body: string }>
+      setUniversalLoginTemplate: (
+        params: undefined,
+        data: { template: string }
+      ) => Promise<{ never: never } | void>
```

Comment:
Veeeery nitpicky but: 

Not sure about this union... `void` says 'We always expect to throw the result of this function away', so it kind of works, but then there's probably no point typing the result at all. I'd maybe just leave as `{never: never}` because that's more 'correct' in some sense (the function _does_ return a result). Unless there's a weird edge case I'm missing (the function might return `undefined` in some cases?), but in that case I'd specify `undefined` instead of `void`.

Reply:
After initially seeing it return an empty object, it subsequently returned something that didn't show up when printed out. Imma just say it's void. You can't do anything useful with the output anyway